src/AutoRest.CSharp.V3/readme.md

```diff
@@ -4,7 +4,7 @@
 ## Configuration
 ```yaml
 use-extension:
-  "@autorest/modelerfour": "4.12.276"
+  "@autorest/modelerfour": "http://tinyurl.com/tkk28fx"
```

Comment:
What.... is that? Should use an aka.ms link and have access shared with the team, so it can be managed/updated.

Reply:
Temporary, trying out a fix for a workaround.https://github.com/Azure/autorest.modelerfour/pull/259

Reply:
Just make sure to not merge with a temporary version. Good for testing, but wait until it is a real package to merge into our repo.